### PR TITLE
Update cloud faq article

### PR DESCRIPTION
**Pull request by alina-tincas**
## Description

_What did you add/update/change?_
Update cloud faq article: styleguide, missing region (australian) + .net versions support -> linked to eol for easier future maintaintence + SQL ce is removed since v9 so left just SQL instead 

- noticed as well that many questions are similar to the ones from https://umbraco.com/products/umbraco-cloud/faq/ so am not sure if an idea could be to remove questions that are on docs and just put a link to .com to read more about them there?💡
- 
## Type of suggestion

* [x] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
cloud

## Deadline (if relevant)

_When should the content be published?_
anytime

### CODE REVIEW

File: umbraco-cloud/frequently-asked-questions.md [hunk 1]
```diff
@@ -1172,9 +1174,9 @@ If you need help with this, don't hesitate to reach out to us and we'll be happy
 
 Yes, you can choose between West Europe, East US, South UK, and East Australia regions.
 
-### Can I move my existing project created on Cloud in the EU region to the US region?
+### Can I move my existing project created on Cloud in the EU region to other regions?
 
-Yes, you can move a project that was created on Umbraco Cloud in the EU region to the US region by following the [migrate between regions guide](getting-started/migrate-between-regions.md).
+Yes, you can move a project that was created on Umbraco Cloud in the EU region to another region by following the [migrate between regions guide](getting-started/migrate-between-regions.md).
```
Comment (alina-tincas) [inline umbraco-cloud/frequently-asked-questions.md:1179]:
```suggestion
Yes, you can move a project that was created on Umbraco Cloud in the EU region to another region. Navigate to the [migrate between regions guide](getting-started/migrate-between-regions.md) for more information.
```

File: umbraco-cloud/frequently-asked-questions.md [hunk 2]
```diff
@@ -1042,7 +1044,7 @@ Haven't found an answer to your question? Many security-related questions are an
 
 ### Does Umbraco Cloud support TLS / HTTPS?
 
-Yes, in fact, Umbraco Cloud provides automatic TLS (HTTPS) certificates for ALL hostnames added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 1 year and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
+Yes, in fact, Umbraco Cloud provides automatic TLS (HTTPS) certificates for ALL hostnames added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
```
Comment (alina-tincas) [inline umbraco-cloud/frequently-asked-questions.md:1047]:
```suggestion
Yes, in fact, Umbraco Cloud provides automatic TLS (Transport Layer Security) /HTTPS certificates for ALL hostnames added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
```

File: umbraco-cloud/frequently-asked-questions.md [hunk 3]
```diff
@@ -947,7 +949,7 @@ If you have questions about how many resources your site is using, then please r
 
 Yes, you can. Please note that Umbraco Cloud also uses Cloudflare for DNS, so you need to enroll your hostname as 'DNS Only' with a CNAME pointing to `dns.umbraco.io`. Once you can see the hostname is marked with 'Protected' under the Project / Hostname subpage you can turn on 'Proxying' for the hostname in your Cloudflare account if you need to use specific Cloudflare features like Page Rules.
 
-Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account and let Umbraco Cloud handle the automatic TLS (HTTPS) certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using support@umbraco.com, before bringing in your own Cloudflare setup.
+Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account and let Umbraco Cloud handle the automatic TLS (HTTPS) certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.
```
Comment (alina-tincas) [inline umbraco-cloud/frequently-asked-questions.md:952]:
```suggestion
Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account. Then let Umbraco Cloud handle the automatic TLS (Transport Layer Security) /HTTPS certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.
```

File: umbraco-cloud/frequently-asked-questions.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ Not currently.
 
 ### Can I move my site from Umbraco Cloud?
 
-Yes, you can. Umbraco Cloud uses the very same Umbraco version that you can download and use on your own. So if you decide that Umbraco Cloud isn’t right for you or your sites then you can clone your site, restore your data locally, and delete your Umbraco Cloud project. We’ll be sad to see you go, but we understand there is a huge variety in requirements so we support and encourage you to choose the best solution for your specific site needs.
+Yes, you can. Umbraco Cloud uses the same Umbraco version that you can download and use on your own. So if you decide that Umbraco Cloud isn’t right for you or your sites then you can clone your site, restore your data locally, and delete your Umbraco Cloud project. We’ll be sad to see you go, but we understand there is a huge variety in requirements so we support and encourage you to choose the best solution for your specific site needs.
```
Comment (alina-tincas) [inline umbraco-cloud/frequently-asked-questions.md:31]:
```suggestion
Yes, you can. Umbraco Cloud uses the same Umbraco version that you can download and use on your own. If you decide that Umbraco Cloud isn’t right for you then you can take some actions on your own. You can clone your site, restore your data locally, and delete your Umbraco Cloud project. We’ll be sad to see you go, but we understand there is a huge variety in requirements so we support and encourage you to choose the best solution for your specific site needs.
```

File: umbraco-cloud/frequently-asked-questions.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ No. It's the same as the latest version of Umbraco that you can download.
 
 ### Can I run my high-traffic site on Umbraco Cloud?
 
-Currently, we have benchmarked a "well-built" site with approximately 50,000 unique visitors per day (\~1.5 million per month) that performs very well. For business-critical, high-traffic sites, we recommend that you look into Umbraco Cloud Professional and Umbraco Cloud Enterprise possibly in combination with a dedicated server.
+Currently, we have benchmarked a "well-built" site with approximately 50,000 unique visitors per day (\~1.5 million per month) that performs well. For business-critical, high-traffic sites, we recommend that you look into Umbraco Cloud Professional and Umbraco Cloud Enterprise possibly in combination with a dedicated server.
```
Comment (alina-tincas) [inline umbraco-cloud/frequently-asked-questions.md:19]:
```suggestion
Currently, we have benchmarked a "well-built" site with approximately 50,000 unique visitors per day (\~1.5 million per month) that performs well. For business-critical, high-traffic sites, we recommend that you look into Umbraco Cloud Professional and Enterprise possibly in combination with a dedicated server.
```

File: umbraco-cloud/frequently-asked-questions.md [hunk 6]
```diff
@@ -1042,7 +1044,7 @@ Haven't found an answer to your question? Many security-related questions are an
 
 ### Does Umbraco Cloud support TLS / HTTPS?
 
-Yes, in fact, Umbraco Cloud provides automatic TLS (HTTPS) certificates for ALL hostnames added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 1 year and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
+Yes, in fact, Umbraco Cloud provides automatic TLS (Transport Layer Security) /HTTPS certificates for **all hostnames** added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
```
Comment (alina-tincas) [inline umbraco-cloud/frequently-asked-questions.md:1047]:
```suggestion
Yes, in fact, Umbraco Cloud provides automatic Transport Layer Security (TLS) /HTTPS certificates for **all hostnames** added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
```

File: umbraco-cloud/frequently-asked-questions.md [hunk 7]
```diff
@@ -1042,7 +1044,7 @@ Haven't found an answer to your question? Many security-related questions are an
 
 ### Does Umbraco Cloud support TLS / HTTPS?
 
-Yes, in fact, Umbraco Cloud provides automatic TLS (HTTPS) certificates for ALL hostnames added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 1 year and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
+Yes, in fact, Umbraco Cloud provides automatic Transport Layer Security (TLS) /HTTPS certificates for **all hostnames** added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
```
Comment (alina-tincas) [inline umbraco-cloud/frequently-asked-questions.md:1047]:
```suggestion
Yes, in fact, Umbraco Cloud provides automatic Transport Layer Security (TLS)/HTTPS certificates for **all hostnames** added to an Umbraco Cloud Project's environment. Umbraco Cloud will automatically renew the certificates, which are issued by Cloudflare. By default, the certificates are valid for 90 days and are then automatically renewed for as long as the hostname is active on Umbraco Cloud.
```

File: umbraco-cloud/frequently-asked-questions.md [hunk 8]
```diff
@@ -947,7 +949,7 @@ If you have questions about how many resources your site is using, then please r
 
 Yes, you can. Please note that Umbraco Cloud also uses Cloudflare for DNS, so you need to enroll your hostname as 'DNS Only' with a CNAME pointing to `dns.umbraco.io`. Once you can see the hostname is marked with 'Protected' under the Project / Hostname subpage you can turn on 'Proxying' for the hostname in your Cloudflare account if you need to use specific Cloudflare features like Page Rules.
 
-Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account and let Umbraco Cloud handle the automatic TLS (HTTPS) certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using support@umbraco.com, before bringing in your own Cloudflare setup.
+Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account. Then let Umbraco Cloud handle the automatic TLS (Transport Layer Security) /HTTPS certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.
```
Comment (alina-tincas) [inline umbraco-cloud/frequently-asked-questions.md:952]:
```suggestion
Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account. Then let Umbraco Cloud handle the automatic Transport Layer Security (TLS)/HTTPS certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.
```

File: umbraco-cloud/frequently-asked-questions.md [hunk 9]
```diff
@@ -28,7 +28,7 @@ Not currently.
 
 ### Can I move my site from Umbraco Cloud?
 
-Yes, you can. Umbraco Cloud uses the very same Umbraco version that you can download and use on your own. So if you decide that Umbraco Cloud isn’t right for you or your sites then you can clone your site, restore your data locally, and delete your Umbraco Cloud project. We’ll be sad to see you go, but we understand there is a huge variety in requirements so we support and encourage you to choose the best solution for your specific site needs.
+Yes, you can. Umbraco Cloud uses the same Umbraco version that you can download and use on your own. If you decide that Umbraco Cloud isn’t right for you then you can take some actions on your own. You can clone your site, restore your data locally, and delete your Umbraco Cloud project. We’ll be sad to see you go, but we understand there is a huge variety in requirements so we support and encourage you to choose the best solution for your specific site needs.
```
Comment (alina-tincas) [inline umbraco-cloud/frequently-asked-questions.md:31]:
```suggestion
Yes, you can. Umbraco Cloud uses the same Umbraco version that you can download and use on your own. If you decide that Umbraco Cloud isn’t right for you then you can take some actions on your own. You can clone your site, restore your data locally, and delete your Umbraco Cloud project. We’ll be sad to see you go. But we understand there are some requirements we might not be able to fulfill. So we support and encourage you to choose the best solution for your specific site needs.
```

**Comment by sofietoft** [comment]
Thanks @alina-tincas ! 💪 

We'll get this reviewed as soon as possible.

File: umbraco-cloud/frequently-asked-questions.md [hunk 10]
```diff
@@ -947,7 +949,7 @@ If you have questions about how many resources your site is using, then please r
 
 Yes, you can. Please note that Umbraco Cloud also uses Cloudflare for DNS, so you need to enroll your hostname as 'DNS Only' with a CNAME pointing to `dns.umbraco.io`. Once you can see the hostname is marked with 'Protected' under the Project / Hostname subpage you can turn on 'Proxying' for the hostname in your Cloudflare account if you need to use specific Cloudflare features like Page Rules.
 
-Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account and let Umbraco Cloud handle the automatic TLS (HTTPS) certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using support@umbraco.com, before bringing in your own Cloudflare setup.
+Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account. Then let Umbraco Cloud handle the automatic Transport Layer Security (TLS)/HTTPS certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.
```
Comment (sofietoft) [inline umbraco-cloud/frequently-asked-questions.md:952]:
```suggestion
Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your own Cloudflare account. This lets Umbraco Cloud handle the automatic Transport Layer Security (TLS)/HTTPS certificates for the hostnames you point to your Umbraco Cloud project. Check with our support team, via chat or using <support@umbraco.com>, before bringing in your own Cloudflare setup.
```

File: umbraco-cloud/frequently-asked-questions.md [hunk 11]
```diff
@@ -963,15 +965,15 @@ HTTP requests headers can be useful for for example multilingual purposes to red
 
 Note, the HTTP requests headers are available on all custom hostnames created through Umbraco Cloud. But not the default hostname for the Umbraco Cloud project such as project.euwest01.umbraco.io.
 
-### What versions of .NET does Cloud support?
+### Which Umbraco versions are available on Umbraco Cloud?
 
-By default, Umbraco Cloud runs all Umbraco version 8 projects on .NET 4.8, Umbraco 9 projects on .NET 5.0, Umbraco 10 projects on .NET 6.0, and Umbraco 11 projects on .NET 7.0.
+All the latest supported versions of Umbraco CMS are available on Umbraco Cloud. See the [Long-term Support & End-of-Life](https://umbraco.com/products/knowledge-center/long-term-support-and-end-of-life/) page for more information.
```
Comment (sofietoft) [inline umbraco-cloud/frequently-asked-questions.md:970]:
```suggestion
All supported versions of Umbraco CMS are available on Umbraco Cloud. See the [Long-term Support & End-of-Life](https://umbraco.com/products/knowledge-center/long-term-support-and-end-of-life/) page for more information.
```

File: umbraco-cloud/frequently-asked-questions.md [hunk 12]
```diff
@@ -981,14 +983,14 @@ We automatically upgrade Cloud projects to the latest patch and minor version of
 
 ### My project didn't receive the auto-upgrade. Why?
 
-When we roll out auto-upgrades to Umbraco Cloud projects the very first thing that happens is a check of all environments on a project. This check will verify whether the environments are responding and don't return an HTTP status error. If the auto-upgrader encounters HTTP status errors on any of the environments during this check, the upgrade process is aborted, and your project will not receive the upgrade.
+When we roll out auto-upgrades to Umbraco Cloud projects the first thing that happens is a check of all environments on a project. This check will verify whether the environments are responding and don't return an HTTP status error. If the auto-upgrader encounters HTTP status errors on any of the environments during this check, the upgrade process is aborted. Then your project will not receive the upgrade.
```
Comment (sofietoft) [inline umbraco-cloud/frequently-asked-questions.md:986]:
```suggestion
When we roll out auto-upgrades to Umbraco Cloud projects the first thing that happens is a check of all environments on a project. This check will verify whether the environments are responding and do not return an HTTP status error. If the auto-upgrader encounters HTTP status errors on any of the environments during this check, the upgrade process is aborted. Your project will not receive the upgrade should this happen.
```

**Review by sofietoft — COMMENTED** [review]
Looks good @alina-tincas ! 💪 

I've added a couple of comments here and there.

**Comment by alina-tincas** [comment]
Looks good 💪

**Comment by sofietoft** [comment]
And to answer the question about also having these questions on Umbraco.com - I think it's OK to have them in both places.
This has been discussed before, and we usually agree to keep two separate FAQs, as they are two very different platforms targeting two different audiences 💪 

Hope it makes sense!
I'll make sure this is merged.

**Pull request merged**